packages/checkbox/src/stories/checkbox.stories.ts

```diff
@@ -0,0 +1,85 @@
+import '../checkbox';
+import { html } from 'lit-html';
+
+export default {
+  title: 'API/Checkbox',
+  component: 'ani-checkbox',
+  argTypes: {
+    label: {
+      control: 'text',
+      description:
+        'Rótulo. O estilo do rótulo não é tratado no componente. Temos um exemplo de como fazer essa customização no Live Demo.',
+      table: {
+        type: { summary: 'string' },
+        category: 'Slot',
+      },
+    },
+    disabled: {
+      control: 'boolean',
+      description: 'Indica se o checkbox está desabilitado.',
+      defaultValue: 'false',
+      table: {
+        type: { summary: 'boolean' },
+        category: 'Propriedades',
+        defaultValue: { summary: false },
+      },
+    },
+    checked: {
+      options: ['true', 'false', 'mixed'],
+      control: { type: 'radio' },
+      description: 'Habilita/Desabilita o checkbox',
```

Comment:
Não acha que esta descrição se confunde com o disable? 
```suggestion
      description: 'Marca e desmarca o checkbox',
```